### ISSUE TITLE
[A11y] Collection Landing - Native Widget - "Copied" button

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1823]
### Operating System

Windows

### Browser

Edge

### Affected page

Collection Show

### Admin user?

Yes

### Issue Description

The button to copy the URL has the accessible name "Copied!".  This doesn't describe the purpose of the button.  I'm not sure the icon describes it, either, so this is probably usability as well as accessibility, but if the icon is well-known for copying a link, the wording "Copied!" is definitely not.

Additionally, the visual "Copied!" indicator shows up on hover and focus instead of only on activation of the button.  I'm betting this is a mistake, but I wonder if it's causing the accessible name to be incorrect.

### Suggested Fix

The button should be named something like "Copy URL" or "Copy URL to clipboard".

The visual "Copied!" indicator is a whole different can of worms.  If we use that on button activation (assuming that gets fixed to not just show up on every focus/hover), SR users need to also be alerted that the link was copied.

**Comment by jechols** [comment]
Update: the "Copied!" text isn't always the accessible name for the button.  In some contexts the accessible name is blank.  Not sure what to make of it, but this button clearly has some fundamental problems.

**Comment by jechols** [comment]
QA: probably needs expertise due to the odd problems and screen-reader-only behaviors

**Comment by CGillen** [comment]
Fixed by #1895

**Comment by jechols** [comment]
This works the first time you tab through the links, but then it's "stuck" on the accessible name being "Copied!" and it announcing the button as "button copied" every time you tab through it. No idea what's going on at this point....
